snmp/datadog_checks/snmp/data/profiles/checkpoint-firewall.yaml

```diff
@@ -10,6 +17,50 @@ device:
 
 sysobjectid: 1.3.6.1.4.1.2620.1.*
 
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "checkpoint"
+      serial_number:
+        symbol:
+          MIB: CHECKPOINT-MIB
+          OID: 1.3.6.1.4.1.2620.1.6.16.3.0
+          name: svnApplianceSerialNumber  # Appliance Serial Number
+          # Example: 1711BA4008
+      version:
+        symbol:
+          OID: 1.3.6.1.4.1.2620.1.6.4.1.0
+          name: svnVersion  # SVN version
+          # Example: R80.10
+          # Reference to R80.10 here https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk111841
+      product_name:
+        symbol:
+          OID: 1.3.6.1.4.1.2620.1.6.1.0
+          name: svnProdName  # Product name
+          # Example: SVN Foundation
+      model:
+        symbol:
+          OID: 1.3.6.1.4.1.2620.1.6.16.7.0
+          name: svnApplianceProductName  # Appliance Product Name
+          # Example: Check Point 3200
+      os_name:
+        symbol:
+          OID: 1.3.6.1.4.1.2620.1.6.5.1.0
+          name: osName  # OS name
+      os_version:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          extract_value: 'Linux\s+\S+\s+([\d.])-'
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
`[\d.]` will match a digit plus any character as `.` is the regex wildcard. If you want to match any dotted number you might need to change the regex to `([\d\.]*\d+)

Reply:
`.` inside square brackets should mean literal ASCII dot (except if I'm missing something)

Reply:
We can possibly use a more precise version `(\d+(?:\d+\.)*)` (see details here https://github.com/DataDog/integrations-core/pull/10968#discussion_r777500828)